yt_dlp/extractor/twitch.py

```diff
@@ -827,6 +829,87 @@ def _real_extract(self, url):
                 f'sorted by {self._SORTED_BY.get(sort, self._DEFAULT_SORTED_BY)}'))
 
 
+class TwitchDirectoryClipsIE(TwitchPlaylistBaseIE):
+    _VALID_URL = r'https?://(?:(?:www|go|m)\.)?twitch\.tv/directory/category/(?P<id>[^/]+)/(?:clips|videos/*?\?.*?)'
+
+    _TESTS = [{
+        # Clips (defaults to 7d)
+        'url': 'https://www.twitch.tv/directory/category/starcraft/clips?range=7d',
+        'info_dict': {
+            'id': 'starcraft',
+            'title': 'starcraft - Clips Top 7D',
+        },
+        'playlist_mincount': 3,
+    }, {
+        'url': 'https://www.twitch.tv/directory/category/minecraft/clips?range=30d',
+        'info_dict': {
+            'id': 'minecraft',
+            'title': 'minecraft - Clips Top 30D',
+        },
+        'playlist_mincount': 3,
+    }]
+
+    Clip = collections.namedtuple('Clip', ['filter', 'label'])
+
+    _DEFAULT_CLIP = Clip('LAST_WEEK', 'Top 7D')
+    _RANGE = {
+        '24hr': Clip('LAST_DAY', 'Top 24H'),
+        '7d': _DEFAULT_CLIP,
+        '30d': Clip('LAST_MONTH', 'Top 30D'),
+        'all': Clip('ALL_TIME', 'Top All'),
+    }
+
+    _PAGE_LIMIT = 20
+
+    _OPERATION_NAME = 'ClipsCards__Game'
```

Comment:
I think the hash for this was lost during conflict resolution from your latest merge commit

Reply:
Fixed and confirmed the test passes and properly returns results when run, e.g.

```
yt-dlp --flat-playlist --print "%(webpage_url)s" "https://www.twitch.tv/directory/category/starcraft/clips?range=7d"
```